Reflect the bootstrapping oversight committee

russellb opened the pull request:
The governance doc discussed an oversight committee, but one had not
yet been created. This change reflects an initial bootstrapping
oversight committee appointed at the launch of the project that will
serve in this function as the project gets started. One of the roles
of this group will be to finalize the governance processes for this
committee for the future.

While in the doc anyway, I did some other minor cleanups - minor
grammatical issues and references to things that don't exist.

Signed-off-by: Russell Bryant <rbryant@redhat.com>

russellb committed:
Reflect the bootstrapping oversight committee

The governance doc discussed an oversight committee, but one had not
yet been created. This change reflects an initial bootstrapping
oversight committee appointed at the launch of the project that will
serve in this function as the project gets started. One of the roles
of this group will be to finalize the governance processes for this
committee for the future.

While in the doc anyway, I did some other minor cleanups - minor
grammatical issues and references to things that don't exist.

Signed-off-by: Russell Bryant <rbryant@redhat.com>

lhawthorn commented on governance.md:
```diff
@@ -11,52 +11,46 @@ InstructLab is made up of several projects that are defined as codebases and ser
-## InstructLab Oversight Committee overview
+## InstructLab Oversight Committee Overview
 
-The Oversight Committee is selected by project Maintainers and consists of 3 to 9 leaders on the InstructLab project; these members will serve to supervise the overall project and its health. It will also consist of a selected chair member who will set agendas and call meetings; these meetings can be public or private at the discretion of the Oversight Committee.
```
I do not think that this paragraph needs to be deleted and, in fact, I would urge us to keep it.

mairin replied:
oh its readded below written a bit differently isnt it

lhawthorn commented on governance.md:
```diff
@@ -11,52 +11,46 @@ InstructLab is made up of several projects that are defined as codebases and ser
-* Making project level decisions, including setting technical policies that apply across all components
-* Resolving escalated project decisions when the team responsible is blocked
+* Making project-level decisions, including setting technical policies that apply across all components
+* Resolving escalated project decisions when the team responsible is blocked.
```
Adding a period here makes this line inconsistent with all others in the list. Please remove it.

lhawthorn commented on governance.md:
```diff
@@ -70,7 +64,7 @@ The election will proceed according to the following process:
 
 1. After a nominated individual(s) agrees to be a candidate for the Oversight Committee, project Maintainers will vote. The voting period will be open for a minimum of three business days. It will remain open until a [supermajority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of project Maintainers has voted. Only current Maintainers of active projects are eligible to vote.
```
Either strike the antithetical (s) or the "a" before nominated.

joesepi replied:
I dont think you need to do either of what @lhawthorn suggests here. The sentence should essentially read "After a nominated individual or individuals" which is what the parenthetical "s" is for.

lhawthorn commented on governance.md:
```diff
@@ -105,7 +99,7 @@ The secondary decision making process is done by voting. The following items mus
 * Adding, archiving, or removing projects (simple majority)
 
-Other decisions may, but do not need to be, called out and put up for decision  on the InstructLab community Slack channel. Directions to join the Slack channel can be found [here](https://github.com/instructlab/community/blob/main/InstructLabSlackGuide.md). In the future, a mailing list will be established. This can be done by anyone at any time. By default, any decisions called to a vote will b ...
+Other decisions may, but do not need to be, called out and put up for decision on the InstructLab community Slack channel. Directions to join the Slack channel can be found [here](https://github.com/instructlab/community/blob/main/InstructLabSlackGuide.md). In the future, a mailing list will be established. This can be done by anyone at any time. By default, any decisions called to a vote will be ...
```
Mailings list have been established. This would be discussed on the community list.

https://github.com/instructlab/community/blob/main/Collaboration.md#email-lists

lhawthorn reviewed (changes requested):
Other than this minor nit feedback, LGTM. Please revise.

mairin commented on governance.md:
```diff
@@ -11,52 +11,46 @@ InstructLab is made up of several projects that are defined as codebases and ser
-At launch, the InstructLab Project will not have an Oversight Committee to avoid unnecessary overhead. After the majority of project Maintainers agree that the project has grown to the point where an Oversight Committee is necessary, project Maintainers will begin establishment. Until the Oversight Committee is constituted, duties of the Oversight Committee will be assumed by project Maintainers.
-
-Except where otherwise noted, decisions should always start at the most local level of project governance. For example, decisions that affect only one project, such as the taxonomy repository and not the `ilab` CLI tool, can  happen within that project. While communication between the different project teams is important as they are all interconnected, minor decisions do not need organization-wid ...
+Except where otherwise noted, decisions should always start at the most local level of project governance. For example, decisions that affect only one project, such as the taxonomy repository and not the `ilab` CLI tool, can happen within that project. While communication between the different project teams is important as they are all interconnected, minor decisions do not need organization-wide ...
```
Maybe clearer to say s/within that project/within the taxonomy project

mairin commented on governance.md:
```diff
@@ -11,52 +11,46 @@ InstructLab is made up of several projects that are defined as codebases and ser
-In some cases, groups of maintainers are responsible for more than one repository, such as `ilab` CLI tool maintainers managing the [taxonomy repository](https://github.com/instructlab/taxonomy).
+Project Maintainers focus on a single codebase, a group of related codebases, a service (for example, a website), or a project to support other projects (such as marketing or community management).
 
-To be considered an _active project Maintainer_, it is required to be associated with at least one active, non-archived project. If only listed on archived projects, they become _emeritus Maintainers_ and are no longer eligible to become an organization Maintainer.
```
Why delete this

russellb replied:
It wasn't on purpose. I went through a round of more significant reformatting and then tried to undo those unnecessary changes. I think I lost a bit of content by accident. Thank you for looking closely!

mairin commented on governance.md:
```diff
@@ -11,52 +11,46 @@ InstructLab is made up of several projects that are defined as codebases and ser
+## InstructLab Oversight Committee Overview
 
-The Oversight Committee is selected by project Maintainers and consists of 3 to 9 leaders on the InstructLab project; these members will serve to supervise the overall project and its health. It will also consist of a selected chair member who will set agendas and call meetings; these meetings can be public or private at the discretion of the Oversight Committee.
+The initial Oversight Committee at the launch of the project was appointed by the founding companies of the InstructLab project. This bootstrap committee will serve until the first election of the Oversight Committee using processes and timing as determined by this group.
```
s/companies/sponsors  ?

russellb replied:
yes, that sounds good, thanks

joesepi reviewed (approved):
LGTM. Thanks.

There is mention of moving things to mailing lists. It might be best to keep that change out of this PR and have someone work on all the places where that change needs to take affect with consistent language and links. Just my 2¢ USD.

russellb commented:
> LGTM. Thanks.
> 
> There is mention of moving things to mailing lists. It might be best to keep that change out of this PR and have someone work on all the places where that change needs to take affect with consistent language and links. Just my 2¢ USD.

I just saw this after I went and made all of the changes to reflect that the mailing lists are used instead of Slack channels. Sorry about that. I do think it's important to make the change though, as our slack trial is up any day now, and these governance-related items should be in a place where history won't be lost.

In addition to addressing other feedback, I simplified references to voting. The doc referred to both simple majority and super majority votes. I simplified it to just be "majority" everywhere for now. If anyone on the oversight committee feels its important to maintain this distinction, I can change them here, or we can go back and make decisions about voting types per topic in a follow-up.

russellb commented:
> In addition to addressing other feedback, I simplified references to voting. The doc referred to both simple majority and super majority votes. I simplified it to just be "majority" everywhere for now. If anyone on the oversight committee feels its important to maintain this distinction, I can change them here, or we can go back and make decisions about voting types per topic in a follow-up.

Actually, I apologize for doing this. I will revert it. I'd like to keep this simple so we can merge it and have more focused PRs that change policy in a more substantial way. I think the mailing list change should be considered more of a trivial update since it already said "we'll have a list in the future."

russellb commented:
I spoke with both @lhawthorn @mairin since they had requested some changes. I've addressed their feedback, so they are good for this to merge.
